src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -89,6 +89,7 @@ const DOCS: { [key: string]: DocItem[] } = {
       summary: 'An interactive button with a range of presentation options.',
       exampleSpecs: {
         prefix: 'button-',
+        exclude: ['button-toggle-appearance', 'button-toggle-exclusive', 'button-toggle-overview']
```

Comment:
@annieyw would it work if he just did `exclude: ['button-toggle-']`?

Reply:
That may not work since we are checking if the example name is present in the exclude list (array includes).
```
  doc.examples =
    exampleNames
      .filter(key => key.match(RegExp(`^${doc.exampleSpecs.prefix}`)) &&
        !doc.exampleSpecs.exclude?.includes(key));
```

For `exclude: ['button-toggle-']` to work we could change the code to:
```
  doc.examples =
    exampleNames
      .filter(key => key.match(RegExp(`^${doc.exampleSpecs.prefix}`)) &&
        !doc.exampleSpecs.exclude?.every(excludeName => key.startsWith(excludeName)));
```

Reply:
Do you want to make that change then? Seems more robust.

I think it should be `some` instead of `every` though. And IIRC not all browsers support `startsWith` so `indexOf(excludeName) === 0` might be a better way to go

Reply:
Changing it to 
```
doc.examples =
    exampleNames
      .filter(key => key.match(RegExp(`^${doc.exampleSpecs.prefix}`)) &&
        !doc.exampleSpecs.exclude?.every(excludeName => key.startsWith(excludeName)));
```
should work